spec/ruby/optional/capi/ext/exception_spec.c

```diff
@@ -36,6 +36,19 @@ VALUE exception_spec_rb_set_errinfo(VALUE self, VALUE exc) {
   return Qnil;
 }
 
+VALUE exception_spec_rb_syserr_new(VALUE self, VALUE num, VALUE msg) {
+  char *cstr;
+  if(msg != Qnil) {
```

Comment:
```suggestion
  if (msg != Qnil) {
```
`tool/lint.sh` would catch this but it doesn't run on `spec/ruby/optional/capi/ext/*.c`, maybe it should if it's not too much trouble